packages/@glimmer/node/lib/serialize-builder.ts

```diff
@@ -39,7 +41,7 @@ class SerializeBuilder extends NewElementBuilder implements ElementBuilder {
 
     if (string === '') {
       return this.__appendComment('%empty%') as any as Simple.Text;
-    } else if (current && current.nodeType === Node.TEXT_NODE) {
+    } else if (current && current.nodeType === TEXT_NODE) {
```

Comment:
simple-dom now has a const enum for this, which will just be replaced with the number and a comment

Reply:
That was what I was hoping but it is not exposed through the public API.

Reply:
https://github.com/ember-fastboot/simple-dom/issues/45